ramalama/cli.py

```diff
@@ -1070,3 +1071,50 @@ def inspect_cli(args):
     args.pull = "never"
     model = New(args.MODEL, args)
     model.inspect(args)
+
+
+def main():
```

Comment:
Now we have two main functions to maintain, not a fan of the duplicate code

Reply:
I agree. That's why I tried to remove bin/ramalama. Linking the entrypoint can't be done to code outside of the python package though.

Reply:
I we need this to fix `pip install .` we can still de-duplicate and have two main()'s just create a common_main() with the common code.